.examples/nginx/docker-compose.yml

```diff
@@ -1,26 +1,40 @@
-db:
-  image: mariadb:latest
-  volumes:
-    - ./mysql/runtime2:/var/lib/mysql
-  environment:
-    - MYSQL_ROOT_PASSWORD
-app:
-  image: matomo:fpm
-  links:
-    - db
-  volumes:
-    - ./config:/var/www/html/config:rw
-    - /home/your-logs-folder:/var/www/html/logs
-  env_file:
-    - ./matomo.env
-web:
-  image: nginx:latest
-  volumes:
-    - ./nginx.conf:/etc/nginx/nginx.conf:ro
-  links:
-    - app
-  volumes_from:
-    - app
-  environment:
-    - VIRTUAL_HOST
+version: '3'
 
+services:
+  db:
+    image: mariadb
+    command: --max-allowed-packet=64MB
+    restart: always
+    volumes:
+      - db:/var/lib/mysql
+    environment:
+      - MYSQL_ROOT_PASSWORD=
+    env_file:
+      - ./db.env
+
+  app:
+    image: matomo:fpm-alpine
+    restart: always
+    links:
+      - db
+    volumes:
+#      - ./config:/var/www/html/config:rw
+#      - ./logs:/var/www/html/logs
+      - matomo:/var/www/html
+    environment:
+      - MYSQL_HOST=db
+    env_file:
+      - ./db.env
+
+  web:
+    image: nginx:alpine
+    restart: always
+    volumes:
+      - matomo:/var/www/html:ro
```

Comment:
What does `:ro` means ? Can you explain it shortly in a README.md?

Reply:
`ro` means read only.
https://docs.docker.com/compose/compose-file/#short-syntax-3